core/src/main/codegen/templates/Parser.jj

```diff
@@ -1661,7 +1663,12 @@ SqlNode SqlUpdate() :
         }
     ]
     [ [ <AS> ] alias = SimpleIdentifier() ]
-    <SET> id = SimpleIdentifier() {
+    (
```

Comment:
We only want this new logic in Babel mode. So suggest moving this to Babel's parserImpls.ftl.

Reply:
How would I go about doing this? Is there a way to inject a code snippet rather than a whole method?

Reply:
Yep, here's an example from Michael's recent change to add support for SEL:
https://github.com/googleinterns/calcite/blob/master/core/src/main/codegen/templates/Parser.jj#L1266

Reply:
Ah I see, so I should just add a boolean flag and wrap the snippet in an if?

Reply:
Yeah I think that could be one way to go about it. Something like parser.allowUpdateFromTable?